### PR TITLE
Fix #2427 : new wms links 

### LINKED ISSUE #2427: Update list of WMS
Complete if necessary the WMS list : 

See: 

 - [x] https://www.so.ch/verwaltung/bau-und-justizdepartement/amt-fuer-geoinformation/geoportal/geodienste/wms-web-map-service/ 
 - [x] http://geo.gr.ch/index.php?id=113 
 - [x] http://www.apps.be.ch/geo/de/angebot-an-geodiensten.html   
 - [x] http://service.lisag.ch/wms?request=getCapabilities for kanton URI 
 - [x] http://www.geolion.zh.ch/opendata there are new datasets available
 - [x] http://www.zg.ch/behoerden/direktion-des-innern/geoportal/geodaten-einbinden/wms Can you double check ? does not work for the free layers
 - [x] http://wms.geoportal.ch:8080/geoserver/AVAI/wms

- [ ] is it possible to add a structure/tree in the pulldown?
* Kantone
 * AG
  * list WMS
 * AI
  * list WMS
etc

* Länder
 * Österreich/Voralberg
   * list WMS 
* Italien / Norditialien
   * list WMS

### PR DESCRIPTION
**Pull request by pfanguin**
Fix #2427

### CODE REVIEW

**Comment by pfanguin** [comment]
jenkins test

File: src/js/ImportWmsController.js [hunk 1]
```diff
@@ -79,6 +82,7 @@
            'https://wms.zh.ch/VeloparkieranlagenZHWMS',
            'https://wms.zh.ch/TBAVMSZHWMS',
            'https://wms.zh.ch/WaldarealZHWMS',
+           'http://wms.zh.ch/WildkarteZHWMS',
```
Comment (oterral) [inline src/js/ImportWmsController.js]: https ?

File: src/js/ImportWmsController.js [hunk 2]
```diff
@@ -72,6 +74,7 @@
            'https://wms.zh.ch/TBAStr3ZHWMS',
            'https://wms.zh.ch/TBAStr2ZHWMS',
            'https://wms.zh.ch/TBAStr1ZHWMS',
+           'http://wms.zh.ch/ZVVZHWMS',
```
Comment (oterral) [inline src/js/ImportWmsController.js]: https?

Comment (oterral) [inline src/js/ImportWmsController.js]: Remove the `,`at the end that breaks jenkins

**Comment by pfanguin** [comment]
it's done

**Comment by oterral** [comment]
@pfanguin  missing uri, zug and geoportal.ch wms servers.

**Comment by oterral** [comment]
@pfanguin see in #2427 for the urls

File: src/js/ImportWmsController.js [hunk 3]
```diff
@@ -91,7 +95,49 @@ goog.provide('ga_importwms_controller');
...
+           'https://wms.geo.gr.ch/richtplan',
+           'https://wms.geo.gr.ch/uebersichtsplan',
+           'http://webdienste.zugmap.ch/basisplan_sw/service.svc/get?REQUEST=getcapabilities&Service=WMS&VERSION=1.3.0',
+           'http://webdienste.zugmap.ch/basisplan_farbig/service.svc/get?REQUEST=getcapabilities&Service=WMS&VERSION=1.3.0',
+           'http://webdienste.zugmap.ch/kkgeo_gws_zg/service.svc/get?REQUEST=GetCapabilities&SERVICE=WMS&VERSION=1.3.0',
+           'http://webdienste.zugmap.ch/Landwirtschaft_Naturschutz/service.svc/get?REQUEST=getcapabilities&Service=WMS&VERSION=1.3.0',
+           'http://webdienste.zugmap.ch/luftbild2011/service.svc/get?REQUEST=getcapabilities&Service=WMS&VERSION=1.3.0',
+           'http://webdienste.zugmap.ch/luftbildplus/service.svc/get?REQUEST=GetCapabilities&Service=WMS&VERSION=1.3.0',
+           'http://webdienste.zugmap.ch/ortsplan/service.svc/get?REQUEST=GetCapabilities&Service=WMS&VERSION=1.3.0',
+           'http://webdienste.zugmap.ch/Zonenplan_WMS/service.svc/get?REQUEST=getcapabilities&Service=WMS&VERSION=1.3.0',
```
Comment (oterral) [inline src/js/ImportWmsController.js]: can you remove the parameters of the url , it should work without.

Comment (oterral) [inline src/js/ImportWmsController.js]: for all zug wms

**Comment by cedricmoullet** [comment]
Tested. Thanks.

**Pull request merged**